src/components/Tab/Tabs.js

```diff
@@ -0,0 +1,41 @@
+import React, {useState} from 'react';
+import { v4 as uuid } from 'uuid';
+import { ThemeProvider } from 'styled-components';
+import {StyledTabs} from './Tabs.styled'
+import themeSettings from './theme'
+import { Tab } from './Tab';
+import { TabContent } from './TabContent';
+
+const TabsList = [  
+    {name: 'Home', text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur condimentum lacus nec ligula faucibus rhoncus. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae;'},
+    {name: 'Profile', text: 'Donec dignissim ultricies felis, eu dictum eros congue in. In gravida lobortis libero nec tempus. Cras rutrum nisl ut leo volutpat rhoncus. Nulla massa nulla, viverra hendrerit laoreet at, tincidunt eu lacus.'},
+    {name: 'Contact', text: 'Donec dignissim ultricies felis, eu dictum eros congue in. In gravida lobortis libero nec tempus. Cras rutrum nisl ut leo volutpat rhoncus. Nulla massa nulla, viverra hendrerit laoreet at, tincidunt eu lacus.', disabled:  true}
+]
+const defaultActive = 'Home';
+
+const Tabs = () => {
+    const [activeTab, setActiveTab] = useState(defaultActive);
+
+    const handleClick = (e) => {
+        e.preventDefault();
+        if(e.target.tagName === 'A') {
+            setActiveTab(e.target.name);
```

Comment:
`e.target` to taka operacja bardziej wykorzystywana w czystym JS, w React moglibyśmy zrobić coś na zasadzie `<Tab onClick={() => handleClick(item) }` wtedy w tej funkcji masz dostęp do tego obiektu `item` i możesz pobrać np.  `name`, aby móc go identyfikować.

Reply:
Zmienione